tests/pipelines/kitctl/template.yaml

```diff
@@ -16,17 +16,10 @@ spec:
     params:
       - name: name
         value: '$(params.name)'
-  - name: setup-data-plane
```

Comment:
Why are we getting rid of dataplane setup from the template?

Reply:
This is just an example pipeline not really used for anything but demo. Current dataplane setup is obsolete and prevents this pipeline from succeeding.

Reply:
>Current dataplane setup is obsolete 

Why is it obsolete, whats the issue with the pipeline? If you don't set node count in the spec it won't provision any nodes.